Components/Trips/Trips.js

```diff
@@ -140,44 +160,60 @@ const Trips = ({ navigation }) => {
           )
         })}
       </ScrollView>
-
-      {/* struggling to make FL work using ScrollVIew for now, not aware of the nuiances right now but it works */}
-      {/* <FlatList 
-        data={mockPreviews}
-        renderItem={(item) => (
-          <TouchableOpacity>
-            <View style={styles.container}>
-              <Text style={styles.name}>{item.name}</Text>
-              <View style={styles.cities}>
-                <Text style={styles.origin}>{item.originAbbrev}</Text>
-                <Text style={styles.destination}>{item.finalDestinationAbbrev}</Text>
-              </View>
-              <View style={styles.cities}>
-                <Text style={styles.startDate}>{item.startDate}</Text>
-                <Text style={styles.endDate}>{item.endDate}</Text>
-              </View>
-            </View>
-          </TouchableOpacity>
-        )}
-      /> */}
     </View>
   )
 }
 
-export default Trips;
-
 const styles = StyleSheet.create({
+  trips: {
+    backgroundColor: '#96CDEC',
+    borderWidth: 1.5,
+    borderColor: 'black',
+    flex: 1,
+    height: 3000,
+    justifyContent: 'center',
+    padding: 20,
+    width: '100%'
+  },
+  header: {
+    borderBottomWidth: 3.5,
+    borderBottomColor: 'black',
+    fontSize: 30,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    marginTop: 3,
+    paddingBottom: 10,
+    width: '100%'
+  },
```

Comment:
Great naming conventions - easy to follow, and builds on general html tag knowledge